myst-management/performance/README.md

```diff
@@ -25,5 +26,36 @@ Don't forget to restart the container.
 docker restart myststudio_web
 ```
 
+
+
+## Tuning Memory parameters
+
+Memory parameters can also be tuned to improve the performance. In order to tune the memory, you can apply the following parameters in the ***docker-compose.yml*** file, as shown below. Restarting Myst would make the changes take effect with the new values. 
+
+```
+    environment:
+      CATALINA_OPTS: "-Xmx3072m -Xms3072m"
+```
+
+(Please be careful of the formatting)
+
+
+
+## Improve Performance using Logging/Log rotation :
+
+Performance can be improved by rotating log files, which can turn huge in size, resulting in MyST's performance degradation. We can apply log-rotation inside the ***docker-compose.yml*** file, as below..
+
+Add the following under the required container, in the above file..
```

Comment:
Where is an example. **Do not** use images as the only example. Users cannot copy text from images.